fastlane/Appfile

```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
+# apple_id("[[APPLE_ID]]") # Your Apple email address
```

Comment:
Should this be filled out? If not, do we need it in source control at all?

Reply:
the values are not needed, we don't build and upload an IPA, we build only an XCFramework

Reply:
the value are marked as comment with # in the beginning, they are template files of fastlane. every fastlene user knows them